src/libs/ajax/goggle-storage-models.ts

```diff
@@ -0,0 +1,7 @@
+export type RequesterPaysErrorInfo = {
```

Comment:
broke out these types/type-predicates to be in a secondary module so that some consumers can just import what they need from here and not pull in the GoogleStorage.ts module.